src/utils/apiRequests/api.ts

```diff
@@ -115,4 +115,19 @@ export async function putAuthenticatedRequest(url:any,data:any,token:any) {
   });
   const result = await res.json();
   return result;
+}
+
+export async function getRequestWithoutRedirecting(url: any) {
+  let result;
+  await fetch(`${process.env.API_ENDPOINT}` + url, {
+      headers: {
+        'tenant-key': `${process.env.TENANTID}`,
+        'X-SESSION-ID': await getsessionId()
```

Comment:
This request does not sent the x-locale header. Do we really need another getRequest?

Reply:
Well, is this special case it does not matter, if we do not show any error raise and returned by the API call.

Reply:
I think x-locale is only required in POST requests. we need to handle API errors in the UI instead of redirecting the user to 404. though, changing this now will require major refactoring so just created another function for now.

Reply:
Ok